activities/member-relations/index.md

```diff
@@ -4,4 +4,76 @@ type: Index
+
+The process towards full membership is roughly divided into three phases, noting that every organization we speak to is unique. This means that individual membership  processes may differ between the organizations we work with.
+
+If you're interested in membership for your organization, or have any questions, please email us at <membership@publiccode.net>
+
+## 1. Membership growth
+
+During membership growth we connect with and support wholly publicly-owned organizations in their path towards membership.
+
+In this phase, we aim to:
+
+* build trust in our organization and our processes
+* determine focal and contact points within the prospective member organization
+* gain insight into the member's organizational structures and how codebase stewardship can benefit them
+
+We conduct various activities during membership growth, like meetings to exchange knowledge and support with contracting. We can also offer workshops to raise awareness about:
+
+  * codebase stewardship
+  * public code
+  * the [Standard for Public Code](https://standard.publiccode.net/)
+  * our organizational model
+
+To make it easier for the potential member, we have regular calls to answer any questions and ensure continuous, open communication.
+
+Membership growth is finalized when the public organization signs a [membership contract](../membership-growth/membership-contract.md).
+
+## 2. Member onboarding
+
+During member onboarding we help new members get accustomed to our processes, and support their public code projects.
+
+In this phase, we aim to:
+
+* support members with their public code projects and codebases
+* introduce members to our network and codebase communities
+* prepare members for their first general assembly
+
+During member onboarding, we usually conduct a [membership workshop](../membership-growth/founding-membership-workshop.md) to:
+  * identify public code projects and codebases to collaborate on
+  * set expectations for membership
+
+We also do other activities that seem useful, for example:
+
+* developing a membership plan to identify the member's support needs, and a timeline of milestones
+* conducting trainings on various topics surrounding public code
+* assessing codebases for codebase stewardship
+
+## 3. Membership
+
+During membership, members become fully involved in governing the Foundation for Public Code and build mature public code projects in their organizations.
+
+In this phase, we aim to:
+
+* Fully support members with all elements of codebase stewardship
+* Build capacity on various topics surrounding public code
+* Empower members to take ownership of steering our processes, governance, and finances
+* Maintain healthy communications amongst members and the Foundation for Public Code team
+
+The following are examples of activities we may conduct with our members:
+
+* Codebase stewardship
+* Building capacity on various topics surrounding public code
+* Brokering collaborative solutions within member community
```

Comment:
This looks gramatically incorrect - does it mean between members (our member community) or is it missing an article and a possessive apostrophe (a member's community)?